website/app/styles/app.scss

```diff
@@ -5,58 +5,58 @@
 
 // HDS COMPONENTS
 
-@import "@hashicorp/design-system-components";
-@import "@hashicorp/design-system-power-select-overrides";
+@use "@hashicorp/design-system-components";
+@use "@hashicorp/design-system-power-select-overrides";
 
 // Tokens (CSS props)
-@import "tokens";
+@use "tokens";
 
 // Global declarations
-@import "global";
-@import "typography";
-@import "spacing";
+@use "global";
+@use "typography";
+@use "spacing";
 
 // "Doc" components
-@import "doc-components";
+@use "doc-components";
 
 // Markdown
-@import "markdown";
+@use "markdown";
 
 // Page-specific declarations
-@import "pages/application"; // main application frame
-@import "pages/home"; // homepage (index)
-@import "pages/about/principles";
-@import "pages/foundations/landing";
-@import "pages/foundations/icon";
-@import "pages/foundations/typography";
-@import "pages/components/accordion";
-@import "pages/components/alert";
-@import "pages/components/app-footer";
-@import "pages/components/application-state";
-@import "pages/components/button";
-@import "pages/components/code-block";
-@import "pages/components/copy-snippet";
-@import "pages/components/copy-button";
-@import "pages/components/dropdown";
-@import "pages/components/flyout";
-@import "pages/components/landing";
-@import "pages/components/modal";
-@import "pages/components/pagination";
-@import "pages/components/radio-card";
-@import "pages/components/rich-tooltip";
-@import "pages/components/side-nav";
-@import "pages/components/super-select";
-@import "pages/components/table";
-@import "pages/components/tabs";
-@import "pages/layouts/app-frame";
-@import "pages/whats-new/changelog";
-@import "pages/patterns/table-multi-select";
-@import "pages/patterns/filter-patterns";
-@import "pages/utilities/dialog-primitive";
+@use "pages/application"; // main application frame
+@use "pages/home"; // homepage (index)
+@use "pages/about/principles";
+@use "pages/foundations/landing" as foundations-landing;
+@use "pages/foundations/icon";
+@use "pages/foundations/typography" as foundations-typography;
```

Comment:
ha! so `foundations/landing` results in the same module name as `components/landing` and `foundations/typography` clashes with the global `typography`.

thought: I now wonder if we should give aliases to all page-specific styles, following the same pattern you used for these

Reply:
I could do that. I was wondering about it but since it didn't actually seem to be needed yet I didn't add aliases for the other pages.

Reply:
So do you think it's helpful if I go ahead and add them?

Reply:
Yes, I would do that. At least it's a more consistent approach

Reply:
Pushed up. Let me know if there's others you think should be given aliases as well.